tests/auxiliary/minicore.rs

```diff
@@ -42,6 +42,9 @@ pub trait Copy: Sized {}
 #[lang = "bikeshed_guaranteed_no_drop"]
 pub trait BikeshedGuaranteedNoDrop {}
 
+#[lang = "freeze"]
+trait Freeze {}
```

Comment:
Question: does it matter that the definition in `core::marker` is currently

https://github.com/rust-lang/rust/blob/23032f31c91f2bb74ba4be20e075fcc929f66527/library/core/src/marker.rs#L862-L864

As in, does it need to also match

```rs
pub unsafe auto trait Freeze {} 
```
?

Reply:
If we make it an auto trait we need to also copy the opt-outs. But I guess now that this is in a central file that's actually worth it.